webviews/createPullRequestViewNew/app.tsx

```diff
@@ -151,6 +151,17 @@ export function main() {
 					create();
 				}
 
+				function activateCommand(event: MouseEvent | KeyboardEvent, command: string): void {
+					if (event instanceof KeyboardEvent) {
+						if (event.key === "Enter" || event.key === " ") {
```

Comment:
We should use single quotes here:
```suggestion
						if (event.key === 'Enter' || event.key === ' ') {
```